tests/cypress/integration/actions_tasks_objects/case_5_image_rotate.js

```diff
@@ -31,7 +32,8 @@ context('Check if the image is rotated', () => {
             .then(($styles) => {
                 scaleAfter = Number($styles.match(/scale\((\d\.\d+)\)/m)[1]);
                 cy.expect(scaleBefore).to.be.greaterThan(scaleAfter);
-                cy.get('#cvat_canvas_content').dblclick();
+                cy.get('#cvat_canvas_content').dblclick().wait(500);
```

Comment:
Why not 300 or 10000?

Reply:
Reduced the waiting time. Also separated this behavior depending on the browser. This time was revealed when checking the passing of the test in CI when using the Firefox browser. At the moment, the Cypress repository has an open issue on the issue of different test results in the Firefox browser in headless mode and without it. https://github.com/cypress-io/cypress/issues/8610
If necessary, can I open a separate issue and specify the CVAT repository to reproduce the error?